perf_test/CMakeLists.txt

```diff
@@ -116,7 +116,7 @@ IF(KokkosKernels_ENABLE_BENCHMARK)
       )
       TARGET_LINK_LIBRARIES(
         ${BENCHMARK_NAME}
-        PRIVATE benchmark::benchmark Kokkos::kokkoskernels
+        PRIVATE benchmark::benchmark Kokkos::kokkoskernels impl_git_version
```

Comment:
Not too keen on this new library idea, can we just add the header to `Kokkos::kokkoskernels` library directly? I am worried that this sets the idea that every new feature can lead a new library in the project, which seems un- calable as a design?